app/models/solr_document.rb

```diff
@@ -29,4 +29,7 @@ class SolrDocument
       self[Solrizer.solr_name(term.to_s)]
     end
   end
+
+  # Fix for error on search page, we aren't using resource type the way Hyrax is
+  def itemtype; end
```

Comment:
I'd suggest we do 

```ruby
def itemtype
  ResourceTypesService.microdata_type((resource_type || []).first))
end
```

[ResourceType#microdata_type](https://github.com/samvera/hyrax/blob/8a9a3fc2f8fa82333f0de0869819aa9a6675b7a5/app/services/hyrax/resource_types_service.rb#L18) handles its `nil` argument case.